Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -527,19 +530,21 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
             i++;
         }
 
-        prevYPosForDraw = yPos;
+        if ([runs objectAtIndex:0] && static_cast<_CTRun*>([runs objectAtIndex:0])->_dwriteGlyphRun.glyphCount != 0) {
```

Comment:
>static_cast<_CTRun*>([runs objectAtIndex:0])->_dwriteGlyphRun.glyphCount != 0 [](start = 38, length = 77)

didn't you have a different fix for this in your other pull request? that one had this part of the if in a smaller block